solutions/common-shared-flows/not-found-v1/sharedflowbundle/policies/Raise.NotFound.xml

```diff
@@ -0,0 +1,8 @@
+<RaiseFault name="Raise.NotFound">
+  <FaultResponse>
+     <Set>
+      <StatusCode>404</StatusCode>
+      <ReasonPhrase>Resource Not Found</ReasonPhrase>
```

Comment:
We should discuss our recommendation for how to populate `<ReasonPhrase/>`. I've so far advised keeping to the list of standard phrases (per RFC 7231) only because I've experienced one [poorly implemented] client that choked hard on a custom phrase.